server/configmodule.py

```diff
@@ -65,6 +65,16 @@ class DevelopmentLiteConfig(Config):
     SCHEME = 'http'
 
 
+class DevelopmentSvObsConfig(Config):
```

Comment:
could you add a note at the top of this file that config.py should be updated if new configs are added?

Reply:
Done